openpathsampling/tests/test_pyemma.py

```diff
@@ -0,0 +1,92 @@
+import openpathsampling as paths
+import numpy as np
+import os
+import pytest
+from itertools import combinations
+from .test_helpers import data_filename
+from openpathsampling.engines.openmm.tools import trajectory_from_mdtraj
+pyemma = pytest.importorskip("pyemma")
+mdtraj = pytest.importorskip("mdtraj")
+
+
+class TestPyEMMA(object):
```

Comment:
```suggestion
class TestPyEMMAFeaturizerCV(object):
```

Could you also rename the file similarly? At least to `test_pyemma_cv.py`; `test_pyemmafeaturizercv`  might be a bit hard to read.